Library/Homebrew/brew.sh

```diff
@@ -810,6 +810,7 @@ fi
 AUTO_UPDATE_COMMANDS=(
   install
   outdated
+  search
```

Comment:
Why is this necessary now?

Reply:
Idea was this is the replacement for remote searching. The code we're removing always worked based on the latest remote state of the taps so doing an auto-update would keep similar behaviour.

Reply:
I'm conflicted on this. I see the benefit of this always being up-to-date. On the flip side, I've heard a bunch of people complain that "`brew search` is too slow" and I fear this may make it worse.

I'm tempted to say we merge without this and then be willing to add it in if we get any complaints or confusion over inconsistencies?

Reply:
Speed shouldn't really be any slower than `brew outdated` after this change, unless the slowness is in DidYouMean or something.

So not really convinced about that as a reason, but I am happy to drop it anyway.

Reply:
Dropped now.